src/plugin/cursus/Entity/Course.php

```diff
@@ -179,6 +179,18 @@ public function getDefaultSession()
         return $defaultSession;
     }
 
+    public function hasAvailableSession()
+    {
+        $now = new \DateTime();
+        foreach ($this->sessions as $session) {
+            if ($session->getEndDate() > $now) {
```

Comment:
do we need to handle the case where `Session::$endDate` is null?

Reply:
Yes we should. Thank you.